addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2741,7 +2741,8 @@ options.registry.SnippetMove = options.Class.extend({
                 }
                 break;
         }
-        if (params.name === 'move_up_opt' || params.name === 'move_down_opt') {
+        if (!this.$target[0].classList.contains('o_we_no_scroll_on_move')
```

Comment:
You should not have to add code here I think, can't you use the data-exclude attribute of the SnippetMove option to prevent that?

Reply:
The goal here is to make scroll effect optional (after moving "Accordion" items), I don't see how we can get this behaviour using "data-exclude" :thinking:

Reply:
Oh, my bad :+1:

Reply:
@xO-Tx In fact, even if I know we did that for other thing before, it is really not ideal to add classes on snippet to tweak editor behavior (this make the website visitor DOM bigger for no reason on their side).

Instead you could add a `data-no-scroll` param on the `SnippetMove` option with `.accordion .card` as value. Then you can check here if the element matches that selector you receive via that param (which can be extended by other snippets or modules).